reconcile/utils/oc.py

```diff
@@ -160,9 +160,10 @@ def __init__(self, namespace, resource, server,
 
 
 class OC:
-    def __init__(self, server, token, jh=None, settings=None,
+    def __init__(self, name, server, token, jh=None, settings=None,
```

Comment:
`name`? That can be anything... Maybe `cluster_name` or `cluster`.

Since you're modifying the constructor for this class, it is a fantastic moment to document its usage <3

Reply:
i love adding docs